packages/flamegrill/src/flamegrill.ts

```diff
@@ -31,15 +31,33 @@ export interface CookResults {
   [scenarioName: string]: CookResult;
 }
 
+
+/**
+ * Function that takes a directory path
+ * and creates the structure if it doesn't exist
+ * @param {string} path directory path
+ * @returns {string} the directory path
+ */
+function resolveDir(dirPath: string): string {
+  mkdirp(dirPath, (err, made) => {
+    if (made === null){
+      console.log("Unable to create directory");
+      console.error(err);
+    }
+  })
```

Comment:
Could Node's `fs.mkdirSync` be used here instead of adding a package?

Reply:
yup seems to work fine will push it now